lib/main.dart

```diff
@@ -36,29 +36,254 @@ class _MyHomePageState extends State<MyHomePage> {
       length: 2,
       child: Scaffold(
         appBar: AppBar(
+          leading: IconButton(
+            icon: const Icon(Icons.airline_seat_individual_suite_rounded),
+            color: Colors.white,
+            onPressed: () {},
+          ),
           backgroundColor: Colors.grey,
           bottom: TabBar(
-            indicator: BoxDecoration(
-                borderRadius: BorderRadius.circular(30),
-                color: Colors.redAccent),
+            indicator: BoxDecoration(borderRadius: BorderRadius.circular(30), color: Colors.redAccent),
             tabs: const [
-              Tab(icon: Icon(Icons.favorite),),
-              Tab(text: "boobs",),
+              Tab(
+                icon: Icon(Icons.favorite),
+              ),
+              Tab(
+                text: "boobs",
+              ),
             ],
           ),
           title: const Text('I'),
         ),
-        body: TabBarView(
-            physics: const NeverScrollableScrollPhysics(),
-            children: [
+        body: TabBarView(physics: const NeverScrollableScrollPhysics(), children: [
           SizedBox(
             child: Image.asset('assets/images/tlo.jpg'),
           ),
           SizedBox(
             child: Image.asset('assets/images/jakob.jpg'),
-          )
+          ),
         ]),
       ),
     );
   }
 }
+
+class SecondPage extends StatefulWidget {
+  const SecondPage({Key? key}) : super(key: key);
+
+  @override
+  _SecondPageState createState() => _SecondPageState();
+}
+
+class _SecondPageState extends State<SecondPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        children: [
+          Stack(
+            alignment: Alignment.topCenter,
+            children: <Widget>[
+              Image.asset('assets/images/warzywa.jpg'),
+              Align(
+                alignment: Alignment.bottomLeft,
+                child: Container(
+                  decoration: const BoxDecoration(
+                    borderRadius: BorderRadius.all(Radius.circular(8.0)),
+                  ),
+                  width: 128,
```

Comment:
czemu podajemy width i height? Ogarnij to paddingami

Reply:
to jest wielkość logo lidla. Jak mam to paddingami zrobić?

Reply:
constrains w Containerze, albo ConstrainedBox, masz na dole w kodziku, który wysłałem to fixnięte

Reply:
jak sobie nie pullnąłeś tego kodu to pullnij